docs/docs/guides/monitoring.md

```diff
@@ -41,8 +41,8 @@ helm install prometheus prometheus-community/kube-prometheus-stack
 ### Customizing Prometheus
 
```

Comment:
I think it is go to have one paragraph:

In this section you can customize the Prometheus chart. These are the important parameters you should change:
- `hosts` under all of the components - so that the components are reachable on your configured ingress host.
- `adminPassword`, under the `grafana` component - will be used to login via the web.
- ...

Reply:
I did some changes have a look if you like it now or feel free to make another suggestion